src/style.css

```diff
@@ -0,0 +1,125 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__photo-Mac {
+  height: 134px;
+  width: 160px;
+}
+
+.card__description {
+  text-align: left;
+  padding-top: 40px;
+  font-family: inherit;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
Use CSS variables for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)